src/main/java/com/direwolf20/buildinggadgets/common/tools/MathTool.java

```diff
@@ -0,0 +1,16 @@
+package com.direwolf20.buildinggadgets.common.tools;
+
+public final class MathTool {
+
+    private MathTool() {
```

Comment:
Why is the a constructor in a class like this?

Reply:
That is standard...
As Effective Java  suggests, utility classes should be made truly uninstantiable amd therefore have an private default constructor.

As you might have noticed, I've been adding these constructors to our utility classes over time...

Reply:
Though this should be according to formatting guidelines be:
```suggestion
    private MathTool() {}
```

(the closing brace in the same line)